lib/backends/kv-backend.js

```diff
@@ -35,6 +35,11 @@ class KVBackend extends AbstractBackend {
           this._logger.warn(`Failed to JSON.parse '${value}':`, err)
           return
         }
+
+        if (!(secretProperty.property in parsedValue)) {
+          throw new Error('Could not find property ' + secretProperty.property + ' in ' + secretProperty.key)
```

Comment:
Thanks for the pul request! Could you add a unit test for this case?

Also, it's probably better to check `parsedValue[secretProperty.property]` to match the rest of the code, or even check the result on line 43 before returning.

Reply:
Not sure I understand... `parsedValue[secretProperty.property]` could contain an empty value, so checking that wouldn't help with verifying if the property exists.